SUTrivBot/Repo/triviaQAData.json

```diff
@@ -1,67 +1,1003 @@
+    {
+      "questionText": "In Steven's dream in Together Alone, name something he's juggling.",
+      "answerText": "MC Bear Bear, Chaaaps, ukulele",
+      "answers": ["MC Bear Bear", "Chaaaps", "ukulele"],
+      "answersRequired": 1,
+      "bonusAnswers": [""],
+      "bonusAll": false,
+      "points": 1,
+      "bonusPoints": 0
+    },
+    {
+      "questionText": "Non-Crystal gems fuse in Together Alone. What gems are they/what is the fusion's name?",
+      "answerText": "(Lemon) Jade",
+      "answers": ["Jade", "lemon"],
+      "answersRequired": 1,
+      "bonusAnswers": [""],
+      "bonusAll": false,
+      "points": 1,
+      "bonusPoints": 0
+    },
+    {
+      "questionText": "What food did Connie luckily grab before Stevonnie was locked up after the ball?",
+      "answerText": "Protes (protein bar)",
+      "answers": ["Protes", "protein bar"],
+      "answersRequired": 1,
+      "bonusAnswers": [""],
+      "bonusAll": false,
+      "points": 1,
+      "bonusPoints": 0
+    },
+    {
+      "questionText": "What's one of the things that helps Steven psychic-project to Earth in Escapsim? ",
+      "answerText": "Beach City, video games, donuts, Dad",
+      "answers": ["Beach City", "video games", "donuts", "Dad"],
+      "answersRequired": 1,
+      "bonusAnswers": [""],
+      "bonusAll": false,
+      "points": 1,
+      "bonusPoints": 0
+    },
+    {
+      "questionText": "What two things finishes off Steven's boat in Escapsim?",
+      "answerText": "giant spear, cape",
+      "answers": ["spear", "cape"],
+      "answersRequired": null,
+      "bonusAnswers": [""],
+      "bonusAll": false,
+      "points": 1,
+      "bonusPoints": 0
+    },
+    {
+      "questionText": "How does Steven defeat the watermelon shark?",
+      "answerText": "by smooching it!",
+      "answers": ["smooching", "kissing"],
+      "answersRequired": 1,
+      "bonusAnswers": [""],
+      "bonusAll": false,
+      "points": 1,
+      "bonusPoints": 0
+    },
+    {
+      "questionText": "What does Steven write to Bismuth and Greg in Escapsim?",
+      "answerText": "HELP! WE NEED BACKUP!",
+      "answers": ["HELP", "NEED", "BACKUP"],
+      "answersRequired": null,
+      "bonusAnswers": [""],
+      "bonusAll": false,
+      "points": 1,
+      "bonusPoints": 0
+    },
+    {
+      "questionText": "Where did Pink Diamond get the organic worms from?",
+      "answerText": "Kyanite Colony",
+      "answers": ["Kyanite Colony"],
+      "answersRequired": null,
+      "bonusAnswers": [""],
+      "bonusAll": false,
+      "points": 1,
+      "bonusPoints": 0
+    },
+    {
+      "questionText": "Tell us a Bismuth pun. Can be from the show or original!",
+      "answerText": "don't know what to do with this one ¯\\_(ツ)_/¯ Maybe this will be a mod-only question",
+      "answers": ["¯\\_(ツ)_/¯"],
+      "answersRequired": null,
+      "bonusAnswers": [""],
+      "bonusAll": false,
+      "points": 1,
+      "bonusPoints": 0
+    },
```

Comment:
I'd clear this from here all-together. 
We can add mod questions as an additional feature later as handling those will be different. 
Allowing mods to switch points around needs to be wired up too. Could have PeriBot message the trivia master directly with the points results for each question and allow the trivia master to edit points through direct message to PeriBot. 

Should also restrict that a user can only be a trivia master of one game at a time. (it requires monitoring and commands so don't want someone distracted trying to run more than one game at a time.)